WooCommerce/Classes/Extensions/Publisher+Validation.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Combine
+
+typealias ValidationPublisher = AnyPublisher<Bool, Never>
+
+class ValidationPublishers {
+    /// Validates whether a string has content other than whitespace and newlines
+    ///
+    static func contentValidation(for publisher: Published<String>.Publisher) -> ValidationPublisher {
+        return publisher.map { value in
+            return value.trimmingCharacters(in: .whitespacesAndNewlines).isNotEmpty
+        }
+        .dropFirst()
+        .eraseToAnyPublisher()
+    }
```

Comment:
I'm not sure I like the idea of hiding away `dropFirst()` in the encapsulation of `ValidationPublishers`. If there are cases where we want to reuse this, we'd need to make sure that we want the same behavior: to ignore the first element of the publisher. I think we can omit this implementation and replace it with something simpler, more information in the next comment.